### PR TITLE
Subsystem/climber

### PR DESCRIPTION
**Pull request by ghost**
Climber Subsystem

### CODE REVIEW

**Review by borkae — CHANGES_REQUESTED** [review]
Sync constants

**Review by duffym22 — CHANGES_REQUESTED** [review]
Sync constants

File: src/main/java/frc/robot/Constants.java [hunk 1]
```diff
@@ -19,7 +19,15 @@
...
-    //Declare all constants in the form: SUBSYSTEM_NAME_ID = x;
+    public static final int CLIMBER_CLIMBER_RIGHT_ID = 10;
+    public static final int CLIMBER_CLIMBER_LEFT_ID = 35;
+    public static final int CLIMBER_CLIMBER_DRIVE_ID = 17;
+    //public static final int CLIMBER_SOLENOID_ID = 1;
+    public static final int CLIMBER_SOLENOID_RIGHT_ID = 0;
+    public static final int CLIMBER_SOLENOID_LEFT_ID = 0;
+
+  
+  //Declare all constants in the form: SUBSYSTEM_NAME_ID = x;
```
Comment (duffym22) [inline src/main/java/frc/robot/Constants.java]: Remove duplicate values from Constants.java as they already exist.

File: src/main/java/frc/robot/subsystems/Subsystem_Climber.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+public class Subsystem_Climber extends SubsystemBase {
+  /**
+   * Creates a new Subsystem_Climber.
+   */
+  TalonSRX climberRight = new TalonSRX(Constants.CLIMBER_CLIMBER_RIGHT_ID);
+  TalonSRX climberLeft = new TalonSRX(Constants.CLIMBER_CLIMBER_LEFT_ID);
+  VictorSPX climberDrive = new VictorSPX(Constants.CLIMBER_CLIMBER_DRIVE_ID);
+  Solenoid solenoidLeft = new Solenoid(Constants.CLIMBER_SOLENOID_LEFT_ID);
+  Solenoid solenoidRight = new Solenoid(Constants.CLIMBER_SOLENOID_RIGHT_ID);
```
Comment (duffym22) [inline src/main/java/frc/robot/subsystems/Subsystem_Climber.java]: Reference existing constants not duplicate ones

**Review by duffym22 — CHANGES_REQUESTED** [review]
See comments. If you need assistance syncing your branch Constants.java - speak with a mentor

**Comment by borkae** [comment]
Replace the CLIMBER_CLIMBER_* constants with the correct ones.

**Comment by ghost** [comment]
Issues #7

**Comment by borkae** [comment]
Looks good.

File: src/main/java/frc/robot/Constants.java (hunk 1 above)
Comment (borkae) [reply src/main/java/frc/robot/Constants.java]: Resolved

**Review by borkae — APPROVED** [review]

**Review by mott-lab — APPROVED** [review]

**Pull request merged**